torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -776,6 +777,95 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
   }
 
  public:
+  // One way to avoid overhead of forming string would be to return
```

Comment:
Currently this returns the hierarchy as a vector of module definitions as mangled <type>::<function_name> (plus some special keys for things like top modules, etc.).

I suggest documenting this as well (i.e. what this function actually returns).